### DISCUSSION TITLE
Dutch

### DISCUSSION THREAD

**Discussion by jkokhuis** [MyFirstBitcoin/.github#16] [Translations]
Discussion area for translation of the Bitcoin Diploma from English to Dutch

**Comment by jkokhuis** [comment]
Hieronder een overzicht van veelgebruikte termen in het boek, inclusief een voorstel voor vertaling.
Jullie feedback is welkom.

English - Dutch
(non)custodial - (non)custodial
51% Attack - 51% Aanval
A store of value - Een opslagplaats van waarde
Acceptability - Aanvaardbaar(heid)
Activity: Class Exercise - Oefening in de klas:
Altcoin Season - Altcoin seizoen
Altcoins - Altcoins
Atomic Swap - Atomic Swap
B-Money - B-Money
Bartering - Ruilhandel
Bit Gold - Bit Gold
Bitcoin Blockchain - Bitcoin Blockchain
Bitcoin ledger - Bitcoin grootboek
Bitcoin Lightning Network - Bitcoin Lightning Netwerk
Bitcoin miners - Bitcoin miners
Bitcoin mining - Bitcoin mining
Bitcoin node - Bitcoin node
Bitcoin wallet - Bitcoin wallet
block - block
Block Explorer - Block Explorer
Block reward - Block reward
Block Subsidy - Block subsidy
blockchain - blockchain
blockchains - blockchains
Broadcast - Broadcast
Capital Controls - Kapitaalbeperkingen
Censorship Resistant - Bestand tegen censuur
Central Bank Digital Currencies - Digitaal centralebankgeld
Centraliseerde systemen - Gecentraliseerde systemen
Centralized Exchanges - Gecentraliseerde exchanges
Circular Economies - Circulaire economieën
Circulating Supply - Circulerend aanbod
Class Discussion - Discussie in de klas
Class Exercise - Oefening in de klas
Closed Source - Closed-source
Closed-Source - Closed-source
Cold Storage - Cold Storage
Commodity - Goederen
Commodity Money - Goederengeld
Consensus Mechanism - Consensusmechanisme
Contractionary fiscal policy - Krimpend begrotingsbeleid
Cryptocurrencies - Crypto valuta
Custodial - Custodial
Custodial Wallets - Custodial Wallets
Cypherpunks - Cypherpunks
Debasement - Geldontwaarding
Debt Burden - Schuldenlast
Decentralized Autonomous Organization - Decentralized Autonomous Organization
Decentralized Currency - Gedecentraliseerd geld
Decentralized Finance - Decentralized Finance
Decentralized ledger - Gedecentraliseerd grootboek
Decentralized Systems - Gedecentraliseerde systemen
Difficulty Adjustment - Difficulty Adjustment
DigiCash - DigiCash
Digital Asset - Digitaal bezit
Digital assets - Digitale bezittingen
Digital currencies - Digitale valuta
Digital money - Digitaal geld
digital signature - digitale handtekening
Distributed Ledger - Gedistribueerd Grootboek
Divisibility - Deelbaar(heid)
Don’t Trust, Verify - Don’t Trust, Verify
Double Coincidence of Wants - Double Coincidence of Wants
Double Spend - Double Spend
Durability - Duurzaam(heid)
DYOR - Doe Je Eigen Onderzoek (DYOR)
e- Gold - e- Gold
E-Cash - E-Cash
e-Gold - e-Gold
Electronic currencies - Digitale valuta
Established History - Gevestigde geschiedenis
Exchange Rate - Exchange Rate
Expansionary fiscal policy - Expansief begrotingsbeleid
Fiat currency - Fiat valuta
Fiat money - Fiat geld
Fiat System - Fiat systeem
FOMO - FOMO
FUD - FUD
Fungibility - Uitwisselbaar(heid)
Gatekeepers - Poortwachters
GDP - BBP
genesis block - genesis block
Genesis-block - Genesis-block
Gross domestic product - Bruto binnenlands product
Halvening - Halvening
halving - halving
halvings - halvings
Hard Fork - Hard Fork
Hash - Hash
Hash Functions - Hash functies
HashCash - HashCash
Hashing - Hashing
HODL - HODL
Hot wallet - Hot wallet
Hyperbitcoinization - Hyperbitcoinization
Initial Coin Offering (ICO) - Initial Coin Offering (ICO)
Key concepts - Begrippen
Layer-2 Protocol - Laag-2 Protocol
Ledger - Grootboek
Legal tender - Wettig betaalmiddel
Legal tender law - Wet op wettig betaalmiddel
Lightning network - Lightning netwerk
Lightning transactions - Lightning transacties
Lightning Wallets - Lightning Wallets
Medium of Exchange - Ruilmiddel
Memory pool - Memory pool
Mempool - Mempool
Merkle Trees - Merkle Trees
Miners - Miners
Mining - Mining
Monetary Inflation - Monetaire Inflatie
Monetary system - Monetaire systeem
Nakamoto - Nakamoto
Nakamoto Consensus Mechanism - Nakamoto-consensusmechanisme
Node - Node
Nodes - Nodes
Non custodial - Non-custodial
Non-custodial - Non-custodial
Non-Fungible Token (NFT) - Non-Fungible Token (NFT)
Off-chain - Off-chain
On-Chain - On-chain
Open Source - Open-source
Open-Source - Open-source
Orphan Block - Orphan Block
P2P - Peer-to-Peer
Payment rails - Betaal infrastructuur
Peer-to-Peer - Peer-to-Peer
Peer-to-Peer Electronic Cash System - Peer-to-Peer Electronic Cash System
Physical money - Fysiek geld
Portability - Overdraagbaar(heid)
Private keys - Privé sleutels
Private-key - Privé sleutel
Private-key cryptography - Cryptografie met privésleutels
Proof-of-Stake (PoS) - Proof-of-Stake (PoS)
Proof-of-Work - Proof-of-Work
Public Keys - Publieke sleutels
Purchasing Power - Koopkracht
Recovery Phrase - Herstelzin
Relay Race - Estafette
Representative money - Representatief geld
satoshis - satoshis
Scarce - Schaars
Scarcity - Schaars(heid)
Seed - Seed
Self custodial - Self-custodial
Self custody - Self-custody
Self-Custodial - Self-custodial
Self-custody - Self-custody
Self-Sovereign - Zelf-soeverein
Sidechain - Sidechain
Smart Contract - Smart Contract
sound - eerlijk
Sound Digital Money - Eerlijk digitaal geld
Sound money - Eerlijk geld
Stablecoin - Stablecoin
Stablecoins - Stablecoins
store of value - waardeopslag
supply schedule - leveringsschema
TCP/IP protocol - TCP/IP protocol
The Lightning Network - Het Lightning Netwerk
Third party - 3e partij
Time preference - Tijdvoorkeur
Time-Stamps - Time-Stamps
Token - Token
Tokenization - Tokenisering
Trading Pair - Handelspaar
Transactions - Transacties
Two-Factor Authentication - 2-Factor authenticatie
Unit of Account - Rekeneenheid
unsound - oneerlijk
Unsound Digital Money - Oneerlijk digitaal geld
Unsound Money - Oneerlijk geld
UTXO - UTXO
UTXO - Unspent Transaction Output - UTXO - Unspent Transaction Output
UTXOs - UTXOs
Verifiable - Verifieerbaar(heid)
Vouchers - Vouchers
Wallet - Wallet
Whale - Whale
White Hat Hacker - White Hat Hacker
Whitepaper - Whitepaper

**Comment by alouldz** [comment]
Arabe
